helm/private-network-ibft-and-orion/besu/values.yaml

```diff
@@ -0,0 +1,46 @@
+# Default values for besu.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+imagePullPolicy: IfNotPresent
+
+image:
+  besu:
+    repository: hyperledger/besu
+    tag: develop
+  orion:
+    repository: pegasyseng/orion
+    tag: develop
+
+
+validators:
+  pvcSizeLimit: "1Gi"
+  pvcStorageClass: "standard"
+  validator1:
+    privateKey: "0x8457b1dc606d05308cc96f604dbde54ad85cb4de508742c8e265080b9e08b48c"
+    publicKey: "0x5d812c3c25ff398ab416968fce9009c2be7ed70a87abc8ea30bd667ce17a9287a6341fbf6ce757bb8148436c39c71296639ea81afcc94cdf908b6e1344f26188"
+
+  validator2:
+    privateKey: "0xa53a0dea5e51a9e0735e331b336af65d60f516c7080a52da683f6fd96342be42"
+    publicKey: "0xb2fba529681ea7f4619556753d40c8689b936fb1c621bc91f94d2938eb58c285d4911457ae4887b9c3bd593b2d608d319c6dc384d6acae2d043a4657029178d3"
+
+  validator3:
+    privateKey: "0x28021485044bf6870160f90b82ab6bbd14c691d4414140a63d047a7e3362c64e"
+    publicKey: "0x00b20ab6a385a2403d64637b3d93cb6d83215a08f29adb6feb4b8bf03387b734444e8b060f53150dea4b9b897823540d19918c13d6f57a5153d190b5fad7bf51"
+
+  validator4:
+    privateKey: "0xfb122a05ab1897ff144e1c9efb0bb3144f1e7f319aa5c55e20ef3d8d8464f4e8"
+    publicKey: "0x5fc1f8dc9f0c03087128e4bd724530e883d7de1a431269876dff9c95b8952f73c7e85ac7b49d85a2ad4950e967319482af435e07a0eab0a98d98449437787a00"
+
+orions:
```

Comment:
Why only 2 key for the 4 orion nodes? With this configuration, it's mean that node 3 and 4 of orion have not load the public key in the `configmap.yaml`. It's what we want for this sample?

Reply:
Yup, fixed